test/index.test.js

```diff
@@ -112,10 +139,55 @@ describe('Probot', () => {
       const robot = probot.load(robot => robot.on('push', spy))
       robot.auth = jest.fn().mockReturnValue(Promise.resolve({}))
 
-      await probot.receive(event)
+      await probot.receive({
+        id: eventMeta.id,
```

Comment:
@bkeepers I’m currently assuming that every event has an `id`. As events might come through different channels then http requests in future, can we still assume they will all have unique IDs as passed in the `x-github-delivery` header for webhook event requests?

Reply:
I made the `.id` property optional